docker-compose.yml

```diff
@@ -3,43 +3,12 @@ version: "3.7"
 services:
   breedbase:
     image: breedbase/breedbase:v0.17
+    restart: always
```

Comment:
This is a good thing for production, though it might be unexpected in development (e.g., a developer who powers down their laptop might not want all containers that were running to automatically restart when they power it back on). Suggest moving the `restart` policy fields to the production.yml.